config_pkg: "cola2_sparus2"
config_folder: config
